### PR TITLE
WIP - Upgrade bunsen dependencies in line with LTS 1.0

### PR DESCRIPTION
**Pull request by rox163**
**This project uses [semver](http://semver.org), please check the scope of this pr:**

- [ ] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [x] #major# - incompatible API change

# CHANGELOG
**upgraded** to ember 2.8, ember-frost-core 1.0.0, ember-prop-types 3.0.0, ember-frost-tabs 5.0.0, ember-frost-popover 4.0.0, ember-frost-fields 4.0.0 and node v6

### CODE REVIEW

**Comment by rox163** [comment]
pending an upgrade to ember-frost-demo-components

File: blueprints/ember-frost-bunsen/index.js [hunk 1]
```diff
@@ -8,13 +8,13 @@ module.exports = {
...
           packages: [
             {name: 'ember-browserify', target: '^1.1.12'},
-            {name: 'ember-bunsen-core', target: '0.12.1'},
-            {name: 'ember-frost-core', target: '0.30.0'},
-            {name: 'ember-frost-fields', target: '^1.0.0'},
-            {name: 'ember-frost-tabs', target: '^3.1.0'},
+            {name: 'ember-bunsen-core', target: '~0.12.0'},
+            {name: 'ember-frost-core', target: '^1.0.0'},
+            {name: 'ember-frost-fields', target: '^4.0.0'},
+            {name: 'ember-frost-tabs', target: '^4.0.0'},
```
Comment (dafortin) [inline blueprints/ember-frost-bunsen/index.js]: There is a PR on `frost-tabs` that will bump it to `5.0.0`. I'm just waiting for the approval.

File: package.json [hunk 2]
```diff
@@ -9,20 +9,19 @@
   },
   "scripts": {
     "build": "ember build",
-    "coverage": "COVERAGE=true ember test -r 'dot'",
```
Comment (dafortin) [inline package.json]: Why are we removing the coverage script?

Comment (dafortin) [inline package.json]: We need to set `COVERAGE=true` in the test script

File: package.json [hunk 3]
```diff
@@ -39,50 +38,50 @@
...
     "broccoli-asset-rev": "^2.4.2",
-    "bunsen-core": "0.18.1",
-    "ember-browserify": "1.1.13",
-    "ember-bunsen-core": "0.12.1",
-    "ember-cli": "^2.7.0",
+    "bunsen-core": "~0.18.2",
+    "ember-browserify": "^1.1.13",
+    "ember-bunsen-core": "~0.12.2",
+    "ember-cli": "~2.8.0",
     "ember-cli-app-version": "^2.0.0",
```
Comment (dafortin) [inline package.json]: Do we need this?

File: package.json [hunk 4]
```diff
@@ -39,50 +38,50 @@
...
     "ember-export-application-global": "^1.1.1",
-    "ember-frost-core": "0.30.1",
+    "ember-frost-core": "^1.0.0",
     "ember-frost-demo-components": "1.0.0",
-    "ember-frost-fields": "1.0.0",
-    "ember-frost-popover": "0.1.0",
-    "ember-frost-tabs": "^3.2.0",
+    "ember-frost-fields": "^4.0.0",
+    "ember-frost-popover": "^4.0.0",
+    "ember-frost-tabs": "^4.0.0",
```
Comment (dafortin) [inline package.json]: same comment as above

File: package.json (hunk 2 above)
Comment (rox163) [inline package.json]: was causing tests to hang like we saw for core. will try to investigate if it is an acceptance test issue again

File: testem.js [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
-  disable_watching: true,
-  framework: 'mocha',
-  launch_in_ci: [
-    'Firefox'
-  ],
-  launch_in_dev: [
+  'framework': 'mocha',
+  'test_page': 'tests/index.html?hidepassed&coverage',
+  'disable_watching': true,
+  'launch_in_ci': [
```
Comment (sandersky) [inline testem.js]: I much prefer to keep this as Firefox so it gets tested in both browsers. I know most devs including myself end up doing development in Chrome and running tests in Chrome so I want the CI testing against Firefox. If you really want the CI to test Chrome then please make it do both.

File: testem.js [hunk 6]
```diff
@@ -1,11 +1,11 @@
 module.exports = {
-  disable_watching: true,
-  framework: 'mocha',
-  launch_in_ci: [
-    'Firefox'
-  ],
-  launch_in_dev: [
+  'framework': 'mocha',
+  'test_page': 'tests/index.html?hidepassed&coverage',
```
Comment (sandersky) [inline testem.js]: We don't need `&coverage` anymore, that was a blanket thing.

File: package.json [hunk 7]
```diff
@@ -113,9 +112,9 @@
   "dependencies": {
     "ember-cli-babel": "^5.1.8",
     "ember-cli-htmlbars": "^1.0.3",
-    "ember-cli-sass": "^5.2.0"
+    "ember-cli-sass": "^5.6.0"
```
Comment (sandersky) [inline package.json]: Why make the minimum version higher, does it not work with `5.2.0` anymore? If that is not the case please keep the version lower.

File: package.json [hunk 8]
```diff
@@ -97,7 +96,7 @@
     "remark-lint": "^5.0.0",
     "sass-lint": "^1.7.0",
     "seamless-immutable": "6.3.0",
-    "svg4everybody": "2.1.0",
+    "svg4everybody": "^2.1.0",
```
Comment (sandersky) [inline package.json]: Please remove this as `ember-frost-core` now provides this via the `vendor` directory.

File: package.json (hunk 4 above)
Comment (sandersky) [inline package.json]: Bunsen uses `ember-frost-tabs` to provide tabs from the root `cells` array in a bunsen view.

File: package.json [hunk 9]
```diff
@@ -39,50 +38,50 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.2",
-    "bunsen-core": "0.18.1",
-    "ember-browserify": "1.1.13",
-    "ember-bunsen-core": "0.12.1",
-    "ember-cli": "^2.7.0",
+    "bunsen-core": "~0.18.2",
+    "ember-browserify": "^1.1.13",
+    "ember-bunsen-core": "~0.12.2",
```
Comment (sandersky) [inline package.json]: Please pin `ember-bunsen-core` and `bunsen-core` to exact versions. I don't want to risk breaking changes in those.

**Comment by dafortin** [comment]
`ember-frost-tabs@5.0.0` was published.

**Comment by dafortin** [comment]
Is this PR ready to go in after the conflict is resolved?

**Comment by sandersky** [comment]
@dafortin no it is not. I need to upgrade a couple other things which I hope to do today.

File: testem.js [hunk 10]
```diff
@@ -1,11 +1,11 @@
 module.exports = {
-  disable_watching: true,
```
Comment (sandersky) [inline testem.js]: I'm not a fan of this file being changed like this. The properties were sorted alphabetically and as far as I can tell for the most part this change just scrambles them into an unsorted manner.

File: package.json [hunk 11]
```diff
@@ -9,20 +9,19 @@
   },
   "scripts": {
     "build": "ember build",
-    "coverage": "COVERAGE=true ember test -r 'dot'",
+    "test": "npm run lint && COVERAGE=true ember test",
```
Comment (sandersky) [inline package.json]: Not sure why you moved test up, the scripts were sorted alphabetically before.

**Comment by sandersky** [comment]
Closing in favor of #277

File: package.json (hunk 11 above)
Comment (rox163) [inline package.json]: I didn't actually manually change any ordering in any file. Big alphabetical fan here. Not sure which merge this came from

**Comment by rox163** [comment]
So did u merge all my work from my fork into your PR?

**Pull request closed without merging**